docs/src/content/docs/reference/plugins.md

```diff
@@ -23,11 +23,17 @@ interface StarlightPlugin {
       config: StarlightUserConfig;
       updateConfig: (newConfig: StarlightUserConfig) => void;
       addIntegration: (integration: AstroIntegration) => void;
+      addRouteMiddleware: (config: {
+        entrypoint: string;
+        order?: 'pre' | 'post';
+      }) => void;
       astroConfig: AstroConfig;
       command: 'dev' | 'build' | 'preview';
       isRestart: boolean;
       logger: AstroIntegrationLogger;
-      injectTranslations: (Record<string, Record<string, string>>) => void;
+      injectTranslations: (dict: {
+        [lang: string]: Record<string, string>;
+      }) => void;
```

Comment:
Curiosity: is this just a readability change to make it more obvious the keys in the top-level object are langs?

Reply:
Ah this was technically a fix: the current docs don’t include the parameter name (`dict: `), just the type which isn’t valid TypeScript. A bit unfortunate that it wraps to be honest.

Reply:
Wow, excellent catch, I totally missed that multiple times 😅

Reply:
Me too 😁